requirements-tests.txt

```diff
@@ -5,3 +5,6 @@ pytest
 pytest-cov
 pytest-timeout
 sphinx
+# mypy testing
```

Comment:
I didn't bother with the suggestion in the former PR, but it's not a useful comment to keep IMO. 
```suggestion
```